### ISSUE TITLE
Load env variables from .env

### ISSUE THREAD

**Issue by chronark** [fauna-brecht/fauna-schema-migrate#18]
You could export them before running any command but I think it would be nice to load them from a .env file for convenience.
Maybe even specify the path and/or variable names in `.fauna-migrate.js` :)

**Comment by fauna-brecht** [comment]
supporting dotenv would probably be most logical right?

**Comment by chronark** [comment]
yeah I think that would be best. If I have some time later, I'll add it but I can't promise.

**Comment by ptpaterson** [comment]
If you've got a repo that uses `fauna-schema-migrate` packed in with app code there could be conflicts with where you would want to stash the variables for just `fauna-schema-migrate`.  Possibly even keeping them separate from serverless-function env variable and also from those exposed to the browser.  

I am thinking of at least CreateReactApp and NextJS apps.  There's like a [whole hierarchy](https://create-react-app.dev/docs/adding-custom-environment-variables/) for which files to use when...

Maybe ".env" located inside the required `fauna` folder would be appropriate?

DOTENV specifically advises against having multiple env files, but when a single repo comprises dev tools, node server code, and browser app code, I feel like it's kindof inevitable.  So conflicts and preferences should be accounted for.

**Comment by fauna-brecht** [comment]
Those are good points @ptpaterson. I might opt to make it configurable as well via the fauna migrate config that is already there. Some users might want to use the same FAUNA admin key and don't want to have two .env files. The default probably should go in the fauna folder. Would also make it easy to generate it initially, considering adding an option in the init to generate that file.

**Comment by MKRazz** [comment]
Maybe there are better variations on this, but I was experimenting with a temporary workaround, adding this script to `package.json`:

```json
  "scripts": {
    "fauna-schema-migrate": "DOTENV_CONFIG_PATH=./.env node -r dotenv/config ./node_modules/@fauna-labs/fauna-schema-migrate/dist/cli.js"
  },
```

Then run with:
```shell
npm run fauna-schema-migrate apply
```
and it will apply the config from a `.env` file at the root of your project. Or change the path to `./fauna/.env` if you want a separate `.env` file at the root of the fauna folder as @ptpaterson [mentioned](https://github.com/fauna-labs/fauna-schema-migrate/issues/18#issuecomment-807405174).
